app/api/media/clean/route.ts

```
@@ -0,0 +1,81 @@
+            }
+
+            cursor = listResult.cursor;
+
+            // Add back-off between pages to prevent overwhelming the system
+            if (cursor) {
+                await new Promise((resolve) => setTimeout(resolve, 200));
+            }
+        } catch (error) {
+            console.error(`Error in batch ${batchCount + 1}:`, error);
+            // Continue with next batch instead of failing completely
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)